client/app/account/page.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+
+import React from 'react'
+import { FormContainer, Loading } from '@/_components/layout'
+import { Button, Heading } from '@/_components/ui'
+import { useGetMeQuery } from '@/_hooks/api'
+import { AccountInfo } from './component/AccountInfo'
```

Comment:
Please import `AccountInfo` through `index.ts` file like the others!

Reply:
Fixed it.
5d6e6c2c328e882a131e0b34ea5966d4983d0586